INSTALL.rst

```diff
@@ -64,6 +65,13 @@ to get the latest version for development.
     poetry config virtualenvs.path ${HOME}/.virtualenvs && \
     poetry run pip install pip==${PIP_REQUIRED_VERSION}
 
+#. Setup local configuration for running docker image::
+
+    cp thenewboston_node/project/settings/templates/.env .
+    vim .env  # edit file if needed
+
+#. Install dependencies, run migrations, etc by doing `Update`_ section steps
```

Comment:
this is not forward compatible, because in general we need settings configured before we can run anything from Django.

I thin we just need to make a special case for NODE_SIGNING_KEY to be configured later